src/main/java/com/solvd/web/GamesPage.java

```diff
@@ -0,0 +1,29 @@
+package com.solvd.web;
+
+import com.solvd.web.components.ProductCard;
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import com.zebrunner.carina.webdriver.gui.AbstractPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class GamesPage extends AbstractPage {
```

Comment:
you don't need to create separate class for this type of page. It's regular ProductPage (SearchPage in your case)